workitem_test.go

```diff
@@ -58,9 +58,11 @@ func TestGetWorkItem(t *testing.T) {
 	payload := app.CreateWorkitemPayload{
 		Type: "system.bug",
 		Fields: map[string]interface{}{
-			"system.title":   "Test WI",
-			"system.creator": "aslak",
-			"system.state":   "closed"},
+			"system.title":          "Test WI",
+			"system.creator":        "aslak",
+			"system.state":          "closed",
+			"system.remote_item_id": "http://github.com/sbose78/project/1",
```

Comment:
Work item can be non-remote also, right? So, do we really need to modify this test?